engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/details/subscription-details.controller.js

```diff
@@ -51,5 +51,7 @@ angular.module('Bastion.subscriptions').controller('SubscriptionDetailsControlle
 
             return "";
         };
+
+        $scope.virtWhoToolTip = translate("If the virt-who field is Yes then the subscription requires the use of virt-who. Learn how to configure and use this tool in the <a href=\"https://access.redhat.com/documentation/en/red-hat-satellite/6.2/single/virtual-instances-guide/\">Virtual Instances Guide</a>.");
```

Comment:
We chatted with the usability folks about this. The main goal is to point out to users who have just loaded a manifest that they need to install and configure virt-who. This is why it is tied to the host subscription. I would suggest we stay in line with the latest downstream comps which show it as "Virt-Who Required" and then linking to the portal docs. @thomasmckay @ehelms